src/main/java/com/microsoft/azure/spring/data/documentdb/core/DocumentDbOperations.java

```diff
@@ -7,21 +7,19 @@
 package com.microsoft.azure.spring.data.documentdb.core;
 
 import com.microsoft.azure.documentdb.DocumentCollection;
-import com.microsoft.azure.documentdb.IndexingPolicy;
 import com.microsoft.azure.documentdb.PartitionKey;
 import com.microsoft.azure.spring.data.documentdb.core.convert.MappingDocumentDbConverter;
 import com.microsoft.azure.spring.data.documentdb.core.query.Query;
+import com.microsoft.azure.spring.data.documentdb.repository.support.DocumentDbEntityInformation;
 
 import java.util.List;
 
 public interface DocumentDbOperations {
 
     String getCollectionName(Class<?> entityClass);
 
-    DocumentCollection createCollectionIfNotExists(String collectionName,
-                                                   String partitionKeyFieldName,
-                                                   Integer requestUnit,
-                                                   IndexingPolicy policy);
+    DocumentCollection createCollectionIfNotExists(DocumentDbEntityInformation information,
+                                                   String partitionKeyFieldName);
```

Comment:
Can we remove the `partitionKeyFieldName` and get partitionKeyFieldName from DocumentDbEntityInformation?

Reply:
Yes, we can. But the impact on IT is more than expected. We may consider to finish it in another PR.